deploy/helm/kubecf/assets/operations/instance_groups/database.yaml

```diff
@@ -212,8 +268,18 @@
 - type: replace
   path: /instance_groups/name=routing-api/jobs/name=routing-api/properties/routing_api/sqldb/username
   value: {{ .Values.features.external_database.databases.routing_api.username | quote }}
+{{ if .Values.features.external_database.ca_cert -}}
+- type: replace
+  path: /instance_groups/name=routing-api/jobs/name=routing-api/properties/routing_api/sqldb/ca_cert?
+  value: |
+    {{- toYaml .Values.features.external_database.ca_cert | nindent 4 }}
+{{- else}}
 - type: remove
   path: /instance_groups/name=routing-api/jobs/name=routing-api/properties/routing_api/sqldb/ca_cert?
+{{- end }}
+- type: replace
+  path: /instance_groups/name=routing-api/jobs/name=routing-api/properties/routing_api/sqldb/skip_ssl_validation?
```

Comment:
Hi, @daviddob I am interested in your changes about external databases. And I have a question about this line. Why do you write the path like:
`/instance_groups/name=routing-api/jobs/name=routing-api/properties/routing_api/sqldb/skip_ssl_validation?`
instead of:
`/instance_groups/name=routing-api/jobs/name=routing-api/properties/skip_ssl_validation?`
Because as far as I know, for routing release, under `sqldb`, there is no `skip_ssl_validation` property:https://github.com/cloudfoundry/routing-release/blob/develop/jobs/routing-api/spec#L74
That's just my understanding, if I am wrong, please correct me, thanks in advance!

Reply:
Great question - the property it should be referencing is [here](https://github.com/cloudfoundry/routing-release/blob/dfef72c0bb70e6c04ad0e52a801efcb9f02d83ac/jobs/routing-api/spec#L127) under `/sqldb/skip_hostname_validation`. Looks like there were a few instances in this PR where I was referencing params from old release versions (i.e. UAA). Just double checked them all just to be sure.

Reply:
I see, thanks for your kind reply!